tools/build/Makefile

```
@@ -4,7 +4,7 @@
 
 requirements_check = true
 gpg_check = false
-go_min_version = 1.12.0
+go_min_version = 1.13
```

shouldn't it be

```suggestion
go_min_version = 1.13.0
```

?

---

there are no 1.13.0 version ends with `.0`
so if ends with `.0` we cannot pass `ifneq ($(go_min_version),$(go_version_check))`
anyway I think `tools/build/Makefile` is out of date and not using now